tools/wltests/test_series

```diff
@@ -786,12 +786,25 @@ match_sha1() {
 	COMMIT_SHA1=$1
 
 	c_info "Current kernel: "
-	CURRENT=$($ADB shell 'uname -a')
-	c_info "   $CURRENT"
+	c_info "   $($ADB shell 'uname -a')"
 
-	[[ $CURRENT = *$COMMIT_SHA1* ]] || return $EAGAIN
+	CURRENT=$($ADB shell 'uname -r | awk -F "-" "{print \$NF}"')
+	CURRENT=${CURRENT:1}
 
-	return $OK
+	[[ $COMMIT_SHA1 =~ $CURRENT* ]] && return $OK
+
+	# It may happen that a tag pointing to a given commit has
+	# a different SHA1 (signed tag), so check if this this is
+        # the case.
+	TAG_NAME=$(git -C $KERNEL_SRC describe --exact-match $COMMIT_SHA1 2> /dev/null) || return $EAGAIN
```

Comment:
nit-pick `s|2> /dev/null|2>/dev/null` (i.e. remove the space)

Reply:
OMG. :)
Good.